tswow-core/Public/TSInstance.h

```diff
@@ -46,6 +46,7 @@ class TC_GAME_API TSInstance : public TSMap {
     TSInstance* operator->() { return this; }
     operator bool() const { return map != nullptr && m_script != nullptr; }
     bool IsNull();
+    void RemoveFromMap(TSPlayer player, bool remove);
```

Comment:
should remove have a default value here?

Reply:
Everywhere I saw it used in map, they always gave a bool value for it. As it's a call for DeleteFromWorld, not to just remove from the map.

Reply:
that's right, maybe we can call it `deleteFromWorld` to make that more clear